src/FreeRTOSConfig.h

```diff
@@ -62,7 +62,7 @@
 #define configTICK_RATE_HZ                      1024
 #define configMAX_PRIORITIES                    (3)
 #define configMINIMAL_STACK_SIZE                (120)
-#define configTOTAL_HEAP_SIZE                   (1024 * 40)
+#define configTOTAL_HEAP_SIZE                   (1024 * 39)
 #define configMAX_TASK_NAME_LEN                 (4)
```

Comment:
Most probably problem with insufficient memory must be handled in some other way. If this is true, please, tell me how I should do it. With old constant build fails..

Reply:
Lowering it to 39 is fine, you do not need to worry